public/js/actions/notifications.js

```diff
@@ -0,0 +1,32 @@
+import * as actionTypes from 'constants/action-types';
+
+
+export function addNotification(text, { type='info', userDismissable=false,
+                                 autoHide=true } = {}) {
+  return (dispatch) => {
```

Comment:
why does this need to be deferred? Won't dispatching an action with `{meta: {delay: ...}}` already defer it in the middleware?